wcfsetup/install/files/lib/system/language/preload/PhrasePreloader.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace wcf\system\language\preload;
+
+use wcf\data\language\Language;
+use wcf\system\language\preload\command\CachePreloadPhrases;
+use wcf\system\WCF;
+
+/**
+ * Provides the URL to the preload cache for
+ * phrases and creates it if it is missing.
+ *
+ * @author Alexander Ebert
+ * @copyright 2001-2022 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core\System\Language\Preload
+ * @since 6.0
+ */
+final class PhrasePreloader
+{
+    /**
+     * Returns the URL to the preload cache. Will implicitly
+     * create the cache if it does not exist.
+     */
+    public function getUrl(Language $language): string
+    {
+        if (!$this->needsRebuild($language)) {
```

Comment:
```suggestion
        if ($this->needsRebuild($language)) {
```

Instead the return value should be inverted within the method. Otherwise the code reads like nonsense.